template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -6,8 +6,9 @@
       "longName": "target-framework-override",
       "shortName": ""
     },
-    "UseLocalDB": {
-      "longName": "use-local-db"
+    "EFProvider": {
+      "longName": "ef-provider",
+      "shortName": "ef"
```

Comment:
I had that originally, then changed it since there are three providers for MySQL I thought that might be confusing.